source/hardware/controllers/native_instruments_traktor_kontrol_s4_mk2.rst

```diff
@@ -122,14 +122,12 @@ Autoslip mode: Holding down :hwlabel:`LOOP RECORDER PLAY` and pressing the :hwla
 Mapping options
 ~~~~~~~~~~~~~~~
 
-If you choose, you can edit the controller script and change the Remix Slot buttons to perform loop rolls instead.
-Also by default, :hwlabel:`SHIFT` + :hwlabel:`CUE` rewinds the track to the beginning but you can change this to a Reverse Roll (or “Censor”) effect instead.
+Settings can be edited in the preference windows, under :guilabel:`Preferences`
+> :guilabel:`Controllers` > :guilabel:`Traktor Kontrol S4 MK2 ...`.
```

Comment:
@ronso0 how does `:guilabel:` work again? I thought we could do :guilabel:`Preferences > Controllers > Traktor Kontrol S4 MK2 ...` and it would print the `>` nicely?

Reply:
I just copied this from the S4 Mk3 manual, so I assume that's how it should normally be done?

Reply:
Just tried it, doing it as one thing does not pring the ">" in a special way, it looks better this way.

Reply:
The current way? Sure.